src/locale/pt-BR.h

```diff
@@ -29,6 +29,7 @@
 #define D_STR_TIMER "Timer"
 #define D_STR_ONTIMER D_STR_TIMER " " D_STR_ON
 #define D_STR_OFFTIMER D_STR_TIMER " " D_STR_OFF
+#define D_STR_TIMERMODE D_STR_MODE " " D_STR_TIMER 
```

Comment:
src/locale/pt-BR.h:32:  Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]

Reply:
Thanks.